Description: relax dependencies
 "image" in Debian doesn't have features "avif" and "exr", so disable them.
Index: czkawka-core/Cargo.toml
===================================================================
--- czkawka-core.orig/Cargo.toml
+++ czkawka-core/Cargo.toml
@@ -74,14 +74,14 @@ version = "0.8"
 version = "2.1"
 
 [dependencies.i18n-embed]
-version = "0.15"
+version = "0.16"
 features = [
     "fluent-system",
     "desktop-requester",
 ]
 
 [dependencies.i18n-embed-fl]
-version = "0.9"
+version = "0.10"
 
 [dependencies.image]
 version = "0.25"
@@ -110,7 +110,7 @@ version = "2.0"
 version = "0.5"
 
 [dependencies.infer]
-version = "0.16"
+version = "0.19"
 
 [dependencies.itertools]
 version = "0.13"
@@ -132,7 +132,7 @@ version = "0.0.4"
 optional = true
 
 [dependencies.lofty]
-version = "0.21"
+version = "0.22"
 
 [dependencies.log]
 version = "0.4.22"
@@ -161,7 +161,7 @@ version = "8.5"
 features = ["debug-embed"]
 
 [dependencies.rusty-chromaprint]
-version = "0.2"
+version = "0.3"
 
 [dependencies.serde]
 version = "1.0"
@@ -190,7 +190,7 @@ version = "0.8"
 features = ["xxh3"]
 
 [dependencies.zip]
-version = "2.2"
+version = ">= 2.2, <6"
 features = [
     "aes-crypto",
     "bzip2",
@@ -213,6 +213,3 @@ heif = [
 #     "image/avif",
 # ]
 libraw = ["dep:libraw-rs"]
-
-[target."cfg(windows)".dependencies.file-id]
-version = "=0.2.1"
